config/gitleaks.toml

```diff
@@ -489,9 +489,6 @@ keywords = [
     "key","api","token","secret","client","passwd","password","auth","access",
 ]
 [rules.allowlist]
-paths = [
-  '''Database.refactorlog'''
-]
```

Comment:
This change is unintentional; it appeared after running `cd cmd/generate/config && go run main.go`. I can remove it if you want

Reply:
@blx this has been resolved